tox.ini

```diff
@@ -15,8 +15,8 @@ usedevelop = True
 install_command = pip install -U {opts} {packages}
 deps =
   -r{toxinidir}/requirements-dev.txt
-  -r{toxinidir}/docs_guidelines/requirements-extra.txt
-commands = sphinx-build -b html -W {posargs} docs_guidelines/ docs_guidelines/_build/html
+  -r{toxinidir}/docs_guide/requirements-extra.txt
+commands = sphinx-build -b html -W {posargs} docs_guide/ docs_guide/_build/html
 
 [testenv:doctest]
 usedevelop = True
```

Comment:
You forgot to change `docs_guidelines` to `docs_guide` here and CI is breaking 😅

Reply:
I thought I have changed it. I had some weird merge conflicts in https://github.com/Qiskit/qiskit_sphinx_theme/pull/166/commits/429348798766ad4960e63f0b725528931870922d so maybe that lost some changes.